src/main/scala/viper/gobra/translator/implementations/translator/BuiltInMembersImpl.scala

```diff
@@ -381,6 +418,152 @@ class BuiltInMembersImpl extends BuiltInMembers {
           in.Access(closedInst, in.FullPerm(src))(src)
         )
         in.Function(x.name, args, Vector(), pres, posts, None)(src)
+
+      case (AppendFunctionTag, Vector(sliceT: in.SliceT, _: in.SliceT)) =>
+        /**
+          * requires forall i int :: 0 <= i && i < len(s) ==> acc(&s[i])
+          * requires forall i int :: 0 <= i && i < len(stuff) ==> acc(&stuff[i], _)
```

Comment:
why do you use wildcard amounts? Wouldn't it be better to add another ghost argument such that full permissions to the variadic arguments could be recovered?